{"uid":"c84a99d4b16f1317","name":"Проверить, есть возможность подписаться на пользователя","fullName":"tests.test_another_users_page.TestAnotherUsersPage#test_of_the_ability_to_subscribe_to_a_user_from_his_profile","historyId":"aadab30dabf9cab65c1dec52963bfc03","time":{"start":1737798818570,"stop":1737798818570,"duration":0},"description":"Подписываемся на пользователя из профиля. Проверка на новом зарегистрированном пользователе","descriptionHtml":"<p>Подписываемся на пользователя из профиля. Проверка на новом зарегистрированном пользователе</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED\n  (Session info: headless chrome=112.0.5615.165)","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3c739562b342312bdee969cde617371c\")>\ndb_connection = <psycopg.Connection [IDLE] (host=rc1a-8gbxfgy3ifeqz75r.mdb.yandexcloud.net port=6432 user=dyanyshev database=staging_round_confirmation) at 0x7f2c83e30c50>\n\n    @pytest.fixture(scope='function')\n    @allure.title('Регистрация пользователя')\n    @allure.step('Регистрация пользователя')\n    def registration_user(driver, db_connection):\n        \"\"\"\n        Фикстура для регистрации нового пользователя в системе.\n    \n        Эта фикстура выполняет полный процесс регистрации нового пользователя,\n        включая заполнение всех необходимых полей и прохождение верификации по SMS.\n    \n        Процесс регистрации:\n        1. Открывает страницу авторизации\n        2. Выбирает регистрацию по SMS\n        3. Генерирует случайный никнейм\n        4. Вводит пароль\n        5. Генерирует случайный номер телефона\n        6. Получает код подтверждения из Database\n        7. Вводит код подтверждения\n        8. Проверяет приветственное сообщение\n        9. Закрывает приветственное видео (если оно есть)\n        10. Пропускает выбор рубрик\n        11. Закрывает окно с предложением листать ленту\n    \n        :param driver: экземпляр WebDriver для управления браузером\n        :param db_connection: экземпляр подключения к базе данных ROUND\n    \n        :return: None\n    \n        Примечание:\n        - Использует faker для генерации случайных данных\n        - Взаимодействует с MetaBase для получения кода подтверждения\n        - Обрабатывает возможные исключения, такие как отсутствие приветственного видео\n        \"\"\"\n        authorization_page = AuthorizationPage(driver)\n        join_page = JoinPage(driver)\n        db = DBRoundConfirmation(driver)\n        assertion = Assertion(driver)\n        rubric_page = RubricPage(driver)\n        main_page = MainPage(driver)\n    \n        # Открываем страницу авторизации и выбираем регистрацию по SMS\n>       authorization_page.open()\n\nfixtures/fixtures_for_registration_tests.py:60: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nbase/base_page.py:21: in open\n    self.driver.get(self.PAGE_URL)\n../local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:393: in get\n    self.execute(Command.GET, {\"url\": url})\n../local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:384: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f2c83e334d0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_NAME_NOT_RESOLVED\\\\n  (Session info: headless chrome=112.0.5615.165)\",\"stacktrace\":\"\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED\nE         (Session info: headless chrome=112.0.5615.165)\n\n../local/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1737798818570,"stop":1737798818571,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Установка соединения с БД","time":{"start":1737798818571,"stop":1737798820056,"duration":1485},"status":"passed","steps":[{"name":"Установка соединения с БД","time":{"start":1737798818571,"stop":1737798820056,"duration":1485},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'db_connection' for <Function test_of_the_ability_to_subscribe_to_a_user_from_his_profile>>"}],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},{"name":"driver","time":{"start":1737798820056,"stop":1737798820527,"duration":471},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Инициализация локаторов","time":{"start":1737798820527,"stop":1737798820527,"duration":0},"status":"passed","steps":[{"name":"Инициализация локаторов","time":{"start":1737798820527,"stop":1737798820527,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'elements' for <Function test_of_the_ability_to_subscribe_to_a_user_from_his_profile>>"}],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},{"name":"setup","time":{"start":1737798820527,"stop":1737798820527,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Регистрация пользователя","time":{"start":1737798820527,"stop":1737798825566,"duration":5039},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED\n  (Session info: headless chrome=112.0.5615.165)\n\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/fixtures/fixtures_for_registration_tests.py\", line 60, in registration_user\n    authorization_page.open()\n  File \"/usr/workspace/base/base_page.py\", line 21, in open\n    self.driver.get(self.PAGE_URL)\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 393, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Регистрация пользователя","time":{"start":1737798820528,"stop":1737798825565,"duration":5037},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_NAME_NOT_RESOLVED\n  (Session info: headless chrome=112.0.5615.165)\n\n","statusTrace":"  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/fixtures/fixtures_for_registration_tests.py\", line 60, in registration_user\n    authorization_page.open()\n  File \"/usr/workspace/base/base_page.py\", line 21, in open\n    self.driver.get(self.PAGE_URL)\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 393, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n  File \"/usr/local/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"driver","value":"<selenium.webdriver.chrome.webdriver.WebDriver (session=\"3c739562b342312bdee969cde617371c\")>"},{"name":"db_connection","value":"<psycopg.Connection [IDLE] (host=rc1a-8gbxfgy3ifeqz75r.mdb.yandexcloud.net port=6432 user=dyanyshev database=staging_round_confirmation) at 0x7f2c83e30c50>"}],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"driver::0","time":{"start":1737798825639,"stop":1737798825844,"duration":205},"status":"passed","steps":[],"attachments":[{"uid":"e133fb61773fd030","name":"page_source","source":"e133fb61773fd030.html","type":"text/html","size":39},{"uid":"2bd30d6c56ed3b12","name":"screenshot","source":"2bd30d6c56ed3b12.png","type":"image/png","size":10592},{"uid":"9cecda295acce52d","name":"browser_logs","source":"9cecda295acce52d.txt","type":"text/plain","size":0}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Установка соединения с БД::fin","time":{"start":1737799206243,"stop":1737799206243,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"Critical"},{"name":"feature","value":"Профиль другого пользователя"},{"name":"tag","value":"regression"},{"name":"tag","value":"another_users_page"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_another_users_page"},{"name":"subSuite","value":"TestAnotherUsersPage"},{"name":"host","value":"e41825b774ba"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_another_users_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":10,"skipped":0,"passed":13,"unknown":0,"total":23},"items":[{"uid":"12a392fc73ec380d","status":"passed","time":{"start":1737757037998,"stop":1737757044849,"duration":6851}},{"uid":"e2b5cce3b15c63bb","status":"passed","time":{"start":1737743922742,"stop":1737743929701,"duration":6959}},{"uid":"21aa7ff58fd9296d","status":"passed","time":{"start":1737651698863,"stop":1737651705785,"duration":6922}},{"uid":"2ae41cd869a996d6","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//input[@name=\"nickname\"]')","time":{"start":1737569139111,"stop":1737569139111,"duration":0}},{"uid":"759432d438ab18a9","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium-browser is no longer running, so ChromeDriver is assuming that Chrome has crashed.)","time":{"start":1737568955067,"stop":1737568955067,"duration":0}},{"uid":"471b4e1eca03e456","status":"passed","time":{"start":1734033812125,"stop":1734033818599,"duration":6474}},{"uid":"2a79e591559fded9","status":"passed","time":{"start":1733936389451,"stop":1733936400282,"duration":10831}},{"uid":"871a5a22d89cb69f","status":"passed","time":{"start":1733864808002,"stop":1733864817502,"duration":9500}},{"uid":"93abcd34dca9f991","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1733859394765,"stop":1733859407150,"duration":12385}},{"uid":"685e137fe2a43d19","status":"passed","time":{"start":1733856035049,"stop":1733856045906,"duration":10857}},{"uid":"db823cab5a39f2c2","status":"passed","time":{"start":1733770830619,"stop":1733770839890,"duration":9271}},{"uid":"b615b745e922b54b","status":"passed","time":{"start":1733765167861,"stop":1733765177367,"duration":9506}},{"uid":"b426e1c45dc8c5aa","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//button[@data-testing=\"after_register_dialog__btn_choose_later\"]')","time":{"start":1733421090315,"stop":1733421090315,"duration":0}},{"uid":"2acc3a60599ffa17","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1726779751430,"stop":1726779763563,"duration":12133}},{"uid":"9eb1ab5594ddbac9","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//div[@id=\"user-info_following\"]//div[contains(@class,\"user-info_count\")]')","time":{"start":1726776360575,"stop":1726776390735,"duration":30160}},{"uid":"62c6d4f8c78397f5","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1724097930696,"stop":1724097964242,"duration":33546}},{"uid":"201aaf8c6c49e71a","status":"passed","time":{"start":1723828085348,"stop":1723828091264,"duration":5916}},{"uid":"52a30e6ae8b205ee","status":"passed","time":{"start":1723750516245,"stop":1723750523234,"duration":6989}},{"uid":"e327ecf4e07219a9","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1723741040085,"stop":1723741052263,"duration":12178}},{"uid":"425460d3b9e98b88","status":"passed","time":{"start":1723663705230,"stop":1723663711802,"duration":6572}}]},"tags":["regression","another_users_page"]},"source":"c84a99d4b16f1317.json","parameterValues":[]}